src/plugins/kibana_usage_collection/server/collectors/usage_counters/integration_tests/rollups.test.ts

```diff
@@ -151,33 +135,69 @@ describe('usage-counters', () => {
   });
 });
 
+async function createTestCounters(internalRepository: ISavedObjectsRepository) {
+  await createCounters(internalRepository, OLD_ISO, [
+    ['domain1', 'a', 'count', 'server', 28],
+    ['domain1', 'b', 'count', 'server', 29, 'one'],
+    ['domain1', 'b', 'count', 'server', 30, 'two'],
+    ['domain2', 'a', 'count', 'server', 31],
+  ]);
+
+  await createCounters(internalRepository, RECENT_ISO, [
+    ['domain1', 'a', 'count', 'server', 32],
+    ['domain1', 'b', 'count', 'server', 33, 'one'],
+    ['domain1', 'b', 'count', 'server', 34, 'two'],
+    ['domain2', 'a', 'count', 'server', 35],
+    ['domain2', 'c', 'count', 'server', 36],
+  ]);
+}
+
+// domainId, counterName, counterType, source, count, namespace?
+type CounterAttributes = [string, string, string, 'ui' | 'server', number, string?];
```

Comment:
nit... how about we define this as an object? When I was reading the usage, I was only guessing what each value meant.

Reply:
I kinda like the compact form of the array, it's much easier to read when defining multiple counters. Example from another test I'm working on:

```typescript
const FIRST_DAY_COUNTERS: CounterAttributes[] = [
  // domainId, counterName, counterType, source, count, namespace?
  ['dashboards', 'aDashboardId', 'viewed', 'server', 10, 'first'],
  ['dashboards', 'aDashboardId', 'edited', 'server', 5, 'first'],
  ['dashboards', 'aDashboardId', 'viewed', 'server', 100, 'second'],
  ['dashboards', 'aDashboardId', 'edited', 'server', 50, 'second'],
  ['dashboards', 'aDashboardId', 'consoleErrors', 'ui', 3, 'first'],
  ['dashboards', 'aDashboardId', 'consoleErrors', 'ui', 9, 'second'],
  ['dashboards', 'list', 'viewed', 'ui', 256, 'default'],
  ['someDomain', 'someCounterName', 'someCounter', 'server', 13, 'first'],
];
```

I can add a comment on top of the definition call, as a quick visual reference.